### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by R3fractor**
For review

### CODE REVIEW

**Comment by StoverS** [comment]
looks good

File: blastoff.c [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#include <stdio.h>
+int main()
+{
+   int i;
```
Comment (defreez) [inline blastoff.c:4]: Declare `i` in for loop initializer to limit scope.

Comment (defreez) [inline blastoff.c:4]: Missing constant

File: blastoff.go [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package main
+
+import "fmt"
+
+func main() {
+	for i := 0; i <= 10; i++ {
```
Comment (defreez) [inline blastoff.go:6]: Missing constant

File: blastoff.go [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+		fmt.Println(10-i)
```
Comment (defreez) [inline blastoff.go:7]: Better to go from 10 down to zero and just print `i` here.

**Review by defreez — COMMENTED** [review]
Functionality is good, but variables don't match lab instructions.  Also use clang-format and go fmt.

**Pull request merged**